backend/qemu.pm

```diff
@@ -918,7 +918,7 @@ sub start_qemu ($self) {
         }
         if ($vars->{NBF}) {
             die "Need variable WORKER_HOSTNAME\n" unless $vars->{WORKER_HOSTNAME};
-            sp('kernel', '/usr/share/qemu/ipxe.lkrn');
+            sp('kernel', -e '/usr/share/ipxe/ipxe.lkrn' ? '/usr/share/ipxe/ipxe.lkrn' : '/usr/share/qemu/ipxe.lkrn');
```

Comment:
I think this still needs the `dependecies.yaml` change?

Reply:
I am not familiar with this part, could I add the package to below part?
```
test_base_requires:
  '%main_requires':
  cpio:
  perl(Benchmark):
  perl(Devel::Cover):
  perl(FindBin):
  perl(Pod::Coverage):
  perl(Test::Fatal):
  perl(Test::MockModule):
  perl(Test::MockObject):
  perl(Test::MockRandom):
  perl(Test::Mock::Time):
  perl(Test::Mojo):
  perl(Test::Most):
  perl(Test::Output):
  perl(Test::Pod):
  perl(Test::Strict):
  perl(Test::Warnings): '>= 0.029'
  qemu: '>= 4.0'
  qemu-tools:
  qemu-x86:
  procps:
  python3-setuptools:
  xterm-console:
  xorg-x11-Xvnc:
  icewm:
  xterm:
```

Reply:
you should not need to apply of those changes as you did not change anything in this area. You just need to ensure that your branch is rebased.

Reply:
Thanks much!

Reply:
@rfan1 Be sure to run `make update-deps`

Reply:
I don't know how to do that, can you please share me some light?

Reply:
Since we couldn't figure out the errors with update-deps, I went ahead and pushed the commit to your branch. If it continues to be an issue we can debug it some other time.